clients/ts/FunctionalTests/ts/ConnectionTests.ts

```diff
@@ -10,6 +10,9 @@ const commonOptions: IHttpConnectionOptions = {
     logger: TestLogger.instance,
 };
 
+// On slower CI machines, these tests sometimes take longer than 5s
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 10 * 1000;
```

Comment:
If a longer timeout is required then it should be automatic for all tests. Is there a common test file included in every test to include this?

Reply:
Because of the way rollup combines modules, I wasn't positive that I could get it in one place so I felt like the easiest quick fix was to set it everywhere. After this PR I'll do some work to find the right place to put this.